### DISCUSSION TITLE
rabbitmq - raspberrypi

### DISCUSSION THREAD

**Discussion by pixie79** [rabbitmq/rabbitmq-server#2670] [Questions] [answered]
Hi,

I have installed rabbitmq using kubernetes on to the RaspberryPI. This appears to be running and I can get to the UI.

However the pod keeps getting killed due to the liveliness/readness checks.

I am using:
```
$ rabbitmq-diagnostics ping
```

When i exec in to the pod I get the following when i run the command:
*:system_limit*

I am wondering if this is due to a lack of provisioned memory or cpu. Can you suggest what the minimum should be?

Thanks

Mark

**Comment by lukebakken** [comment, marked as answer]
Followed up here: https://groups.google.com/g/rabbitmq-users/c/U-DfDtLM5pw
